From 3f46c3c1176553576fe438009bda55f0ad0780f9 Mon Sep 17 00:00:00 2001
From: "Christopher A. Oliver" <caowasteland@gmail.com>
Date: Tue, 27 Oct 2015 16:45:38 -0400
Subject: [PATCH 0645/1936] Suppress phase randomness slider for PADsynths.

It doesn't do anything to the generated sample even in discrete mode.
---
 src/UI/OscilGenUI.fl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/UI/OscilGenUI.fl b/src/UI/OscilGenUI.fl
index 8dbbf71a..51f40a41 100644
--- a/src/UI/OscilGenUI.fl
+++ b/src/UI/OscilGenUI.fl
@@ -252,7 +252,7 @@ class OscilEditor {open : {public PresetsUI_}
             label rnd
             callback {}
             tooltip {Oscilator Phase Randomness: smaller than 0 is "group", larger than 0 is for each harmonic} xywh {145 290 100 10} type {Horz Knob} box NO_BOX labelsize 10 align 5 minimum -64 maximum 63 step 1
-            code0 {(void)o->init("Prand");//if (oscil->ADvsPAD) o->hide();}
+            code0 {(void)o->init("Prand"); if (!adnotep) o->hide();}
 	    class Fl_Osc_VSlider
           }
           Fl_Group {} {open
-- 
2.47.0

